### ISSUE TITLE
Where is the (en_US)LICENSE / (en_GB)LICENCE file?

### ISSUE THREAD

**Issue by SlySven** [laurent22/simple-gettext#4]
The files' headers suggest this library is licensed under the GPL:

    /*
      Copyright (C) 2010 Laurent Cozic. All right reserved.
      Use of this source code is governed by a GNU/GPL license that can be
      found in the LICENSE file.
    */ 

however there is not such a file in the repository!

IIRC GitHub can pick up on the presence of such a file and include a reference to the licence type on the right of the "# contributors" data on the banner on the "Code" tab:

![screenshot_20171008_190007](https://user-images.githubusercontent.com/6163092/31319429-3d6cd27e-ac5b-11e7-85d1-d15897e86179.png)

This is important to state because the project I am working on uses a GPL 2.0 (or Later) but if your library carries a "GPL 3.0 (or Later)" license then we would have to release under our binaries of our project also under the "GPL 3.0 (or Later)" terms if we decided to make use of your code - as least as I understand it... 😕 

Update: the GitHub [licensing a repository](https://help.github.com/articles/licensing-a-repository/) link seems to be all about the GitHub banner thingy... 😀

**Comment by laurent22** [comment]
I have put a LICENSE file in the repository now (MIT, which I think is more permissive than GPL). It overrules any other comment in any of the files.

**Comment by SlySven** [comment]
Yeah, the English part is the [MIT licence](https://opensource.org/licenses/MIT) so I'll mark this as closed - thanks for dealing with this so quickly.  By the way - where did that French translation come from?

Reason for asking that is that, as it happens, I'm adding internationalisation to a GPL project but we use some MIT licensed libraries already and I was looking for translations of that (although it is not clear that we could __replace__ the English ones with a country specific one - we'd have to keep them and display an "unofficial" translation _alongside_ them)...

**Comment by laurent22** [comment]
Yes that's an unofficial translation and only the English text is relevant for license purposes. I've had this license in French for another project. I've initially copied it from somewhere else but not sure where.

**Comment by SlySven** [comment]
⚠️ Oh, hang-on, you may intend the LICENCE file to over ride anything in the source files but to be honest that just confuses things - you should update the file headers to reflect the change.

However: most importantly __you cannot change the licence from a *GPL* one to the more permissive *MIT* one _without the permission of the other contributor(s)_ whose contribution were (presumably) made under the stated GPL__ (at the top of the files which they modified) - luckily in this case it is only one person (who has been around recently) i.e. @Crayon2000 but they have to okay it - IMHO but IANAL...

**Comment by Crayon2000** [comment]
For what it's worth, I'm OK with any decision @laurent22 takes. As long as the licence is the same in all files :)

**Comment by SlySven** [comment]
Right - so that makes things simple 😉 and with an addition of:

    Copyright © 2017 {Cryon2000's name}

to all the modified files (um, that is all of them 🙂 )  and ideally the MIT text or perhaps just a reference to the {though it does not actually say it is in the file} MIT licence then everything will be perfect.

**Comment by laurent22** [comment]
Thank you @Crayon2000 for the confirmation. I've now updated all the files.
